src/Components/ExternalResult/ResultItem.tsx

```diff
@@ -41,8 +41,8 @@ export default function ResultItem(props: any) {
   }
 
   return (
-    <div className="mt-4">
-      <PageTitle title={"Result details"} />
+    <div>
+      <PageTitle title={"Result details"}  className="px-6 mb-2"/>
```

Comment:
to fix this.
old: 
![image](https://user-images.githubusercontent.com/21201812/94952155-247acb00-0503-11eb-9cf8-b271ed92fc13.png)

new:
![Uploading image.png…]()